loadenv.js

```diff
@@ -35,4 +35,5 @@ module.exports = {
     JSON.parse(process.env.GOOGLE_APPLICATION_CREDENTIALS),
   testAccounts:
     process.env.TEST_ACCOUNTS && JSON.parse(process.env.TEST_ACCOUNTS),
+  slackWebhookUrl: process.env.SLACK_WEBHOOK_URL || ""
```

Comment:
환경변수와 변수명이 더 구체적이어도 좋을 것 같아요!
(신고 접수를 위한 슬랙 웹훅이라는 정보가 포함되면 좋을 것 같습니다 😄)

```suggestion
  slackWebhookUrl: { report: process.env.SLACK_REPORT_WEBHOOK_URL || "" }
```

Reply:
앞으로 확장성 면에서 훨씬 좋겠네요! 수정하겠습니다 :)